Sources/Kommunicate/Classes/KMConversationViewController.swift

```diff
@@ -735,6 +736,19 @@ extension KMConversationViewController {
         })
     }
 
+    private func topMostViewController() -> UIViewController? {
+        var topController: UIViewController? = UIApplication.shared.windows.first?.rootViewController
+        while let presentedViewController = topController?.presentedViewController {
+            topController = presentedViewController
+        }
+        if let navigationController = topController as? UINavigationController {
+            return navigationController.topViewController
+        } else if let tabBarController = topController as? UITabBarController {
+            return tabBarController.selectedViewController
+        }
+        return topController
+    }
```

Comment:
don;t we have existing function to get this ?

Reply:
Updated the old code for extracting of topViewController and used that.